Gamma_Analysis.py

```diff
@@ -310,6 +334,26 @@ def acquire_files():
     return sample_measurements, sample_names
 
 
+def save_peak(sample, energy):
+    cwd = os.getcwd()
+    sample_name = os.path.splitext(sample.filename)[0]
+    sample_folder = os.path.join(cwd, sample_name)
+    # if folder exists, skip next step. Otherwise, create folder for PNGs
+    if not os.path.exists(sample_folder):
+        try:
+            os.makedirs(sample_folder)
+        except OSError:
+            pass
+    label = sample_name + '_' + str(energy) + '_peak'
+    fwhm = 0.05 * (energy)**0.5
+    energy_range = [(energy - 8 * fwhm), (energy + 8 * fwhm)]
+    # generate plot PNG using Spectrum_Peak_Visualization
+    spv.plot_peaks(sample, title=label, energy_range=energy_range)
```

Comment:
Add `plt.savefig(...)` here and give it the full path and you don't need `shutil.move` anymore.